modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -345,4 +373,13 @@ protected void doInitialise() throws InitialisationException {
       throw new InitialisationException(e, this);
     }
   }
+
+  private byte getSourceTransactionalAction() {
+    try {
+      return toActionCode(sourceAdapter.getTransactionalAction());
+    } catch (MuleException e) {
+      LOGGER.error("An error occurred obtaining the Source Transactional Action, defaulting to [NONE]");
+      return ACTION_NONE;
```

Comment:
why are we doing this instead of failing? the cases where an exception is thrown inside the `getTransactionalAction` occur because the value assigned was invalid, and we (should) always fail when that happens. Here we are saying "You used an expression, that's not supported, so we'll just ignore the transactional part"